pptx_blueprint/__init__.py

```diff
@@ -5,6 +5,11 @@
 __Pathlike = Union[str, pathlib.Path]
 
 
+class TemplateOvewriteError(Exception):
```

Comment:
Maybe create issue to introduce library specific base class, so that "typical" users can just catch this base class, and power users / internal users can catch these specific errors

Reply:
Resolving this since it's not really the scope of this PR

Reply:
Agreed. I will actually wait until we introduce another library-specific exception before opening the issue for a base class.